### DISCUSSION TITLE
Does useAnimatedProps memoize results?

### DISCUSSION THREAD

**Discussion by booker-dragon** [software-mansion/react-native-reanimated#1993] [Q&A]
Hi, I am using useAnimatedProps for `d` in SVG Path

As the path is very complex, may I know if `useAnimatedProps` will memoize the results?

**Comment by mrousavy** [comment]
I believe all adaptors (that includes `useAnimatedProps`) memoize results if they are shallow-equal (which is the case for strings). @karol-bisztyga can you confirm?

**Reply by karol-bisztyga** [reply]
cc @piaskowyk
